cobbler/actions/buildiso/__init__.py

```diff
@@ -0,0 +1,243 @@
+"""
+Builds bootable CD images that have PXE-equivalent behavior for all Cobbler distros/profiles/systems currently in
+memory.
+"""
+
+# SPDX-License-Identifier: GPL-2.0-or-later
+# SPDX-FileCopyrightText: Copyright 2006-2009, Red Hat, Inc and Others
+# SPDX-FileCopyrightText: Michael DeHaan <michael.dehaan AT gmail>
+
+import logging
+import os
+import os.path
```

Comment:
What is the reason to have separate import for `os.path`? Importing `os` should also make `os.path` accessible, am I wrong?